src/sundials/sundials_stepper_impl.h

```diff
@@ -0,0 +1,61 @@
+/* -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2024, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------*/
+
+#ifndef _SUNDIALS_STEPPER_IMPL_H
+#define _SUNDIALS_STEPPER_IMPL_H
+
+#include <sundials/sundials_core.h>
+#include <sundials/sundials_stepper.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct SUNStepper_Ops_* SUNStepper_Ops;
+
+struct SUNStepper_Ops_
+{
+  SUNStepperEvolveFn evolve;
+  SUNStepperOneStepFn onestep;
```

Comment:
Is the convention for struct members snake case? E.g., `one_step`

Reply:
Yes but historically the members in an ops function have been one word, all lower case.